src/HexUtils.js

```diff
@@ -40,6 +40,15 @@ class HexUtils {
     return HexUtils.add(hex, HexUtils.direction(direction));
   }
 
+  static neighbours(hex) {
```

Comment:
Nice addition. This got me thinking would it be nice to have (separate) utils to get neighbours for existing hexagon on a existing grid. Of course this can be kind of used for that too.